### DISCUSSION TITLE
Logging improvements - v0.0.8

### DISCUSSION THREAD

**Discussion by aturri** [aturri/ha-zcsazzurro#12] [Announcements]
[![Downloads for this release](https://img.shields.io/github/downloads/aturri/ha-zcsazzurro/v0.0.8/total.svg)](https://github.com/aturri/ha-zcsazzurro/releases/v0.0.8)

## Notable Changes

* Improved logging in order to allow catching auth errors when ZCS response is not a parsable JSON
* Improved logging in order to avoid disclosing device serial numbers
* Tested also on next release 2023.06.1

<hr /><em>This discussion was created from the release <a href='https://github.com/aturri/ha-zcsazzurro/releases/tag/v0.0.8'>Example title - v0.0.8</a>.</em>

**Comment by uninc80** [comment]
Ciao,
posso chiederti come fare ad ottenere i codici di autenticazione:
 auth_key: xxx
  client_code: xxx
C'è un indirizzio Zucchetti a cui scrivere?

Grazie 1000

**Reply by aturri** [reply]
Ciao,
Io ho chiesto a service.azzurrozcs@zcscompany.com

**Reply by Hollerith98T** [reply]
Ho appena provato ma la procedura adesso è un po diversa, cito testualmente:
"Questo indirizzo e-mail non è attivo, per inviare una richiesta di supporto al nostro team la preghiamo di fare riferimento alla pagina ASSISTENZA che può trovare sul sito [www.zcsazzurro.com](http://www.zcsazzurro.com/)."
